src/server/pkg/testutil/mock_pachd.go

```diff
@@ -1257,12 +1282,24 @@ func NewMockPachd() (*MockPachd, error) {
 		return nil, err
 	}
 
+	go func() {
+		mock.errchan <- server.Wait()
```

Comment:
Probably not a big deal since this is in tests, but I would expect this to be a `select` on the `ctx.Done` channel as well. I guess maybe that's not needed because if `ctx.Done` does get closed we're guaranteed to get an error back from this channel?

Reply:
So this doesn't do a select because the `server` is already set to cancel on the `ctx.Done`.  I also thought it would be simpler to always send something on the errchan so upstream listeners would not need to `select` as well.  I'd prefer to keep it this way unless you insist.